.github/workflows/nodejs.yml

```diff
@@ -100,15 +100,13 @@ jobs:
       matrix:
         node-version: [14.x, 16.x, 18.x]
 
-    services:
-      rippled:
-        image: natenichols/rippled-standalone:latest
-        ports:
-          - 6006:6006
-        options: --health-cmd="wget localhost:6006 || exit 1" --health-interval=5s --health-retries=10 --health-timeout=2s
-
     steps:
       - uses: actions/checkout@v3
+
+      - name: Run docker in background
+        run: |
+          docker run --detach --rm --name rippled-service -p 6006:6006 --volume "${{ github.workspace }}/.ci-config/":"/config/" --health-cmd="wget localhost:6006 || exit 1" --health-interval=5s --health-retries=10 --health-timeout=2s --env "ENV_ARGS=-a --start" --env GITHUB_ACTIONS=true --env CI=true xrpllabsofficial/xrpld:latest
```

Comment:
`--rm` removes the container from the machine after it stops running as a cleanup step. Not entirely sure it's necessary in CI, but is definitely useful to avoid clutter locally (wasn't sure if it would just keep adding containers over time without it or not)

Reply:
Fairly certain that Github Actions cleans up everything properly, but it doesn't hurt to leave that in